src/internals/serializable.ts

```diff
@@ -20,6 +20,9 @@ import * as R from "remeda";
 import { extractClassName } from "@/serializer/utils.js";
 import { SerializerError } from "@/serializer/error.js";
 import { Cache } from "@/cache/decoratorCache.js";
+import { startMetricNodeSdkReader } from "@/instrumentation/sdk.js";
+
+startMetricNodeSdkReader();
```

Comment:
Why the start function is here in `serializable` module?

Reply:
Each exported module inherits from this class.